index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="author" content="Emma Ntihinyuka" />
+    <meta name="Description" content="This is a portfolio project" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <ul class="navcontainer">
+        <li><a href="#" class="navbar-logo font">My logo</a></li>
+        <nav>
+          <li class="navbar-toggle">
+            <img src="img/Union.png" alt="menubar" />
+          </li>
+        </nav>
+      </ul>
+    </header>
+    <section class="container">
+      <main>
+        <h1 class="headerone font">I’m Emmanuel Glad to see you!</h1>
+        <p class="text font">
```

Comment:
- [ ] You did a great job 😃 at to design, but as you see here when I try to move the page it is moving with me with a scroll horizontally, so please try to fix this scroll from the page to make your design for the visitors 😊

> Check below to take a look 😊⤵️

| Your Design |
| --- |
|  ![Recording 2023-02-15 at 19 22 05](https://user-images.githubusercontent.com/83360636/219105954-3bba9dc2-54fd-43a2-8e67-0b72e301ca54.gif)    |

Reply:
Done